lightningd/plugin_control.h

```diff
@@ -0,0 +1,14 @@
+#ifndef LIGHTNING_LIGHTNINGD_PLUGIN_CONTROL_H
+#define LIGHTNING_LIGHTNINGD_PLUGIN_CONTROL_H
+#include "config.h"
+#include <lightningd/lightningd.h>
+
+/**
+ * A plugin command which permits to control (start, stop, ..) plugins
+ */
+struct command_result *json_plugin_control(struct command *cmd,
```

Comment:
Not need to make this public; can make `static` in the `plugin_control.c` file.

Reply:
Ok, is an emty header file OK ?

Reply:
Should be okay.